Modules/_winapi.c

```diff
@@ -2806,6 +2806,11 @@ _winapi__mimetypes_read_windows_registry_impl(PyObject *module,
         if (err == ERROR_FILE_NOT_FOUND) {
             err = ERROR_SUCCESS;
             continue;
+        } else if (err == ERROR_ACCESS_DENIED) {
+            /* Skip keys we can't access, as we do on the fallback WinReg implementation */
+            PyErr_SetFromWindowsErr(err);
```

Comment:
`err` needs to be casted, to satisfy actions. I think this will suffice:
```suggestion
            PyErr_SetFromWindowsErr((int) err);
```

Reply:
Thanks for the review!

You're right. I accepted your commit request